src/modules/universal-discovery/components/search/search.pagination.component.js

```diff
@@ -4,6 +4,8 @@ import PropTypes from 'prop-types';
 import './css/search.pagination.component.css';
 
 const SearchPaginationComponent = (props) => {
+    console.warn('[DEPRECATED] SearchPaginationComponent is deprecated, it will be removed from ezplatform-admin-ui-modules');
```

Comment:
something like:
```js
console.warn('[DEPRECATED] SearchPaginationComponent is deprecated');
console.warn('[DEPRECATED] it will be removed from ezplatform-admin-ui-modules 2.0');
console.warn('[DEPRECATED] use ContentTablePaginationComponent instead');
```